doc/io_chaos.md

```diff
@@ -32,57 +77,51 @@ spec:
     cron: "@every 10m"	
 ```
 
-For more sample files, see [examples/io-mixed-example.yaml](../examples/io-mixed-example.yaml). You can edit them as needed. 
+> For more sample files, see [examples](../examples). You can edit them as needed. 
 
-## Usage
+Description: 
 
-### Configuration
+* **selector**: is used to select pods that are used to inject chaos actions.
 
-#### Annotations
+* **action**: represents the IO chaos actions. Currently the **delay**, **errno**, and **mixed** actions are supported. You can go to [*IO chaos available actions*](#io-chaos-available-actions) for more details.
+* **mode**: defines the mode to run chaos actions. Supported mode: `one` / `all` / `fixed` / `fixed-percent` / `random-max-percent`.
+* **duration**: represents the duration of a chaos action. The duration might be a string with the signed sequence of decimal numbers, each with optional fraction and a unit suffix, such as `"300ms"`, `"-1.5h"` or `"2h45m"`.
+* **delay**: defines the value of IO chaos action delay. The duration might be a string with the signed sequence of decimal numbers, each with optional fraction and a unit suffix, such as `"300ms"`, `"-1.5h"` or `”2h45m”`. Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", and "h".
+  If `Delay` is empty, the operator will generate a value for it randomly.
+* **errno**: defines the error code that is returned by an IO action. It is an int32 string like `"32"`. This field need to be set when you choose an `errno` or `mixed` action. If `errno` is empty, the operator will randomly generate an error code for it. You can set the `errno` by referring to [Errors: Linux System Errors](https://www-numi.fnal.gov/offline_software/srt_public_context/WebDocs/Errors/unix_system_errors.html).
```

Comment:
For specific interfaces please look up please refer to its man manual ,such as read(2): http://man7.org/linux/man-pages/man2/read.2.html

Reply:
Commonly used： EINVAL， EIO，ENOMEM,